docs/about.md

```diff
@@ -12,7 +12,8 @@ You can find our main repository [on GitHub](https://github.com/jellyfin/jellyfi
 
 The core team of Jellyfin consists of:
 
-[Andrew](https://github.com/nvllsvm)  
-[Joshua](https://github.com/joshuaboniface)  
-[Vasily](https://github.com/JustAMan)  
-[Anthony](https://github.com/anthonylavado)  
+[Andrew Rabert (nvllsvm)](https://github.com/nvllsvm)
```

Comment:
Why repeat nicks if they're in the link? And if you repeat then maybe do so for Joshua and Anthony as well...

Reply:
One doesn't see the link hence why I repeated the links. Many people know me by nvllsvm or nullsum.